hello-cdep/README.md

```diff
@@ -0,0 +1,63 @@
+Hello-libs
+=========
+hello-cdep is an Android sample that demos [CDEP](https://github.com/google/cdep) native lib distribution system usage.
+Refer to [CDEP documentation](https://github.com/google/cdep/tree/master/doc)
```

Comment:
Could you change the CDep documentation linke to (https://github.com/google/cdep/blob/master/README.md)?